zimbra/src/main/java/fr/openent/zimbra/worker/ICalRequestWorker.java

```diff
@@ -161,8 +165,7 @@ public void getICal(ICalTask task) {
         UserUtils.getUserInfos(this.eb, userId, user -> {
             calendarService.getICal(user)
                 .compose(ical -> {
-                    ical = ical.replaceAll("\\\r\\\n", "\n");
```

Comment:
no longer need to reformat ical
previously used the following : ical = ical.replaceAll("\\\r\\\n ", "");